app/views/pages/privacy-policy.html.haml

```diff
@@ -0,0 +1,72 @@
+    %p
+      Information published in job listings may be passed on to third party job listing sites that use our job listing data via an application programming interface (API), as described in our Terms and Conditions. We are not responsible for how a third party handles any personal information you choose to include in a job listing.
+
+    %h2.heading-medium Why our use of your personal data is lawful
+    %p
+      In order for our use of your personal data to be lawful, we need to meet one (or more) conditions in the data protection legislation. For the purpose of this project, this processing is necessary to complete a task in the public interest as stated under GDPR Article 6 (1)(e).
+
+    %h2.heading-medium Who we will make your personal data available to
+    %p
+      We sometimes need to make personal data available to other organisations. These might include contracted partners (who we have employed to process your personal data on our behalf) and/or other organisations (with whom we need to share your personal data for specific purposes).
+    %p
+      Where we need to share your personal data with others, we ensure that this data sharing complies with data protection legislation. We share specific types of personal data with dxw, who have been contracted by DfE to manage the delivery of the digital service. If you represent a school and are publishing a job listing through Teaching Jobs we share your email with #{link_to('Microsoft','https://privacy.microsoft.com/en-us/privacystatement')}, solely for the purpose of enabling sign-in to the publishing environment.
+
+    %p
+      The firm dxw uses your data data in the following way
+      %ul.list.list-bullet
+        %li for user research that informs development of the service: if you share your name, email address and phone number dxw will store these in its research library
+        %li for security purposes: dxw collects part of the IP addresses of people visiting the Teaching Jobs website (for example, if we were receiving continuous direct denial of service attacks from an IP address range then we could block it)
+        %li to enable users to use the service and receive updates: dxw collects the emails of those publishing job listings on Teaching Jobs
+
+    %h2.heading-medium How long we will keep your personal data
+    %p Neither DfE nor dxw will keep your personal data longer than we need it for the purpose(s) of research, security and/or delivery of the service, and in any case not longer than 7 years. Your data will be deleted when DfE and dxw no longer need it for these purposes, or when 7 years have passed, whichever comes first.
+    %p Partial IP addresses will be deleted after 7 days unless their request caused an error, in which case they remain in our logs for 30 days.
+    %p If you share personal information in a job listing we will retain this while the job vacancy is live and for one year after it has expired.
```

Comment:
@tahb is this something we need a card for?

Reply:
We do need to record it but perhaps Ellie/Fiona/Isobel can take care of tracking this for us to ensure it's done for a years time because our backlog is pretty big!